django/db/backends/postgresql/operations.py

```diff
@@ -265,9 +278,7 @@ def sequence_reset_sql(self, style, model_list):
                             style.SQL_TABLE(qn(model._meta.db_table)),
                         )
                     )
-                    # Only one AutoField is allowed per model, so don't bother
-                    # continuing.
-                    break
```

Comment:
If I revert this, no tests fail

Reply:
It should fail, did you run `test_sequence_reset_sql_statements` ?